content/2016.01.loadtesting.with-oauth-part-2.rst

```diff
@@ -0,0 +1,264 @@
+How to load test a OAuth-protected HTTP API? (Part 2)
+#####################################################
+
+:lang: en
+:date: 2016-01-05
+:url: load-testing-a-http-api-which-uses-oauth-for-authentication-part-2
+:summary: This is part 2 of a series on loadtesting oauth APIs explaining how to test the performance of an HTTP API when its access depends on an external OAuth service.
+
+This is part 2 of a series on loadtesting oauth APIs.
+
+Introduction
+============
+
+`In the previous article </en/load-testing-a-http-api-which-uses-oauth-for-authentication>`_
+we have seen that:
+
+- We would create the OAuth Bearer Token before the load-test start.
```

Comment:
I suggest "should" may be better than "would" in this case, to capture the notion that it's the right thing to do based on discussion the previous article.